src/stack-hci/azext_stack_hci/aaz/latest/stack_hci/cluster/_create.py

```diff
@@ -75,6 +75,24 @@ def _build_arguments_schema(cls, *args, **kwargs):
 
         # define Arg Group "Identity"
 
+        _args_schema = cls._args_schema
+        _args_schema.type = AAZStrArg(
+            options=["--type"],
+            arg_group="Identity",
+            help="Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).",
+            enum={"None": "None", "SystemAssigned": "SystemAssigned", "SystemAssigned, UserAssigned": "SystemAssigned, UserAssigned", "UserAssigned": "UserAssigned"},
+        )
+        _args_schema.user_assigned_identities = AAZDictArg(
+            options=["--user-assigned-identities"],
```

Comment:
Could you please follow this guideline [managed_identity_command_guideline.md](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md) to design the managed identity related parameters?